.github/workflows/cd.yml

```
@@ -0,0 +1,169 @@
+# CD 워크플로우 이름
+name: CD
+
+# 트리거 조건: main 브랜치에 코드가 푸시될 때 실행
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)